src/main/java/meldexun/nothirium/mc/integration/FluidloggedAPI.java

```diff
@@ -0,0 +1,7 @@
+package meldexun.nothirium.mc.integration;
+
+import net.minecraftforge.fml.common.Loader;
+
+public class FluidloggedAPI {
+    public static final boolean isInstalled = Loader.isModLoaded("fluidlogged_api");
```

Comment:
Missing empty line before first element and after last element of class.

Wrong indention: Spaces are used instead of tabs.

Like the other variables that hold information if another mod is loaded this should be moved into the Nothirium class.